clients/bigquery/bigquery.go

```diff
@@ -78,9 +79,13 @@ func (s *Store) PrepareTemporaryTable(tableData *optimization.TableData, tableCo
 	return s.putTable(context.Background(), tableData.TopicConfig.Database, tempTableName, rows)
 }
 
+func (s *Store) IdentifierFor(topicConfig kafkalib.TopicConfig, table string) types.TableIdentifier {
+	return NewTableIdentifier(s.config.BigQuery.ProjectID, topicConfig.Database, table)
+}
+
 func (s *Store) ToFullyQualifiedName(tableData *optimization.TableData, escape bool) string {
```

Comment:
With the exception of S3, all the `ToFullyQualifiedName` methods are the same. Will look at pulling them out in a subsequent PR.

Reply:
(S3's does not use an `escape` argument)